static/app/components/group/assignedTo.tsx

```diff
@@ -61,10 +61,12 @@ function AssignedTo({group, projectId}: AssignedToProps) {
                     data-test-id="assigned-avatar"
                     actor={assignedTo}
                     hasTooltip={false}
-                    size={20}
+                    size={24}
                   />
                 ) : (
-                  <IconUser size="md" />
+                  <IconWrapper>
```

Comment:
🤔 damn do we need it

Reply:
yeah the sizing was broken because of using svg inside flexbox

Reply:
or at least wrapping it in a div fixed the issue, and then it wasn't aligned so it needed to have display flex